data/pycon-apac-2015/videos/marsface-project-detecting-pseudo-artificial-pycon.json

```diff
@@ -0,0 +1,25 @@
+{
+  "summary": "",
+  "speakers": [
+    "Hideki Tanaka"
+  ],
+  "source_url": "https://www.youtube.com/watch?v=XsinyMyoZPU",
+  "description": "R2 DAY2-04 Marsface Project: Detecting Pseudo-artificial Structures on Mars\nSpeaker: Hideki Tanaka\n\nThe discovery of \"Face on Mars\" by NASA in 1976 has attracted widespread popular and scientific interest over the past few decades. In this project, we apply various object detection techniques, such as conventional haar-like feature based methods and state-of-the-art deep learning methods, in order to detect such pseudo-artificial structures on Mars and other planets. The scientific Python ecosystem plays a crucial role in the exploration.\n\nAbout the speaker\n\nHideki Tanaka is a PyData.Tokyo organizer, optical design engineer, photographer based in Tokyo. He is a machine learning and deep learning enthusiast who is addicted to data science competitions.\n\n個人網頁連結 https://www.facebook.com/HidekiPhoto\nTwitter @atelierhide\n組織/公司 PyData.Tokyo\n頭銜 Organizer\n\nhttps://tw.pycon.org/2015apac/zh/program/71",
+  "title": "Marsface Project: Detecting Pseudo-artificial Structures ... - Hideki Tanaka (PyCon APAC 2015)",
+  "thumbnail_url": "https://i.ytimg.com/vi/XsinyMyoZPU/sddefault.jpg",
+  "quality_notes": "hd",
```

Comment:
We're using `quality_notes` to denote issues with the quality. Things like "no sound", "picture cuts out 3:15 in", etc.